internal/config/config.go

```diff
@@ -384,6 +384,7 @@ func peerFromCR(p metallbv1beta2.BGPPeer, passwordSecrets map[string]corev1.Secr
 
 	var nodeSels []labels.Selector
 	for _, s := range p.Spec.NodeSelectors {
+		s := s // so we can use &s
```

Comment:
Neat. This is not causing bugs as we are using &s in pure functions and not carrying it around, but might bite us in the future.

Reply:
yeah, reinforcing the code :-)
could argue the other way too (adding bullshit, making unnecessary copy, etc etc)
but it's the easiest way for the code scanner to see there are no issues here